packages/jest-snapshot/src/__tests__/plugins.test.ts

```diff
@@ -5,13 +5,18 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-beforeEach(() => jest.resetModules());
```

Comment:
I went on fixing tests of this package because of this line. Not sure if it works currently.